.changeset/shy-bobcats-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": major
+---
+
+`SideNav` - renamed `extraBefore/After` generic containers to `ExtraBefore/After` (uppercase `E`)
```

Comment:
Non-blocker: we tend to add an extra paragraph on what to do when migrating ('To migrate...') most of them (as it is this one) are pretty straightforward, but may be valuable to make these instructions explicit.